ui-v2/tests/test-helper.js

```diff
@@ -1,11 +1,9 @@
 import Application from '../app';
 import config from '../config/environment';
 import { setApplication } from '@ember/test-helpers';
-import { start } from 'ember-qunit';
```

Comment:
Does removing this cause any problems? I think `ember exam` will pass through to ember-qunit but I am not sure if normal `ember test` would cause issues

Reply:
Ohhhh, maybe yes, sorry, let me double check all this

Reply:
I tried this out and all my tests run fine locally without this also.